client/src/main/java/com/paypal/selion/reports/runtime/SaverFileSystem.java

```diff
@@ -131,8 +131,8 @@ public PageContents getScreenshotByName(String name) throws IOException {
     @Override
     public void init() {
         logger.entering();
-        (new File(outputFolder)).mkdirs();
```

Comment:
do we have same behaviour when we run locally through testng plugin?

On Mon, Jul 27, 2015 at 5:34 PM, ysecaran <notifications@github.com> wrote:

> In
> client/src/main/java/com/paypal/selion/reports/runtime/SaverFileSystem.java
> <https://github.com/paypal/SeLion/pull/131#discussion_r35528450>:
>
> > @@ -131,8 +131,8 @@ public PageContents getScreenshotByName(String name) throws IOException {
> >      @Override
> >      public void init() {
> >          logger.entering();
> > -        (new File(outputFolder)).mkdirs();
>
> @fakrudeen78 <https://github.com/fakrudeen78> the outputfolder points to
> target/surefire-reports/ which gets created by surefire plugin. It is used
> as the base directory to store all the details of reports. This statement
> can be removed because the outputfolder is already created.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paypal/SeLion/pull/131/files#r35528450>.
>